restapi.module

```diff
@@ -387,8 +387,9 @@ function restapi_delivery_callback($response) {
 
   // For requests in the browser, we'll try to show the output in a more
   // pleasant way.
-  $accept = $request->getHeaderLine('Accept');
-  if ($accept) {
+  $accept      = $request->getHeaderLine('Accept');
+  $is_download = _restapi_is_download_response($response);
```

Comment:
feels like this may not be necessary, what's happening on L395? We're basically saying if the best is text/html then treat it as such, but why is that showing up as best if it's binary / attachment, etc..?

Reply:
The response content type is not taken into account when determining how to respond. So, generally speaking, all browser requests will return text/html, but other clients may specify to see application/json instead, and in that case, the Negotiator will determine to return JSON.

Reply:
Hmm, can we simplify this then? Something like:

````php
$is_download = stripos($request->getHeaderLine('Content-Disposition'), 'attachment') !== FALSE;
````